ocfweb/docs/docs/services/web/jekyll.md

```diff
@@ -0,0 +1,67 @@
+[[!meta title="Jekyll"]]
+
+[Jekyll](https://jekyllrb.com) is a popular static website generator.
+
+## Set up
+
+1. Go to our [web-based SSH client](https://ssh.ocf.berkeley.edu/) and sign in
+   with your username and password. Or, at terminal, [SSH](https://www.ocf.berkeley.edu/docs/services/shell/) into tsuanami.ocf.berkeley.edu
```

Comment:
we don't want to link to the page directly, instead use the intrawiki link syntax, e.g. see how we link to the services/shell page here https://github.com/ocf/ocfweb/blame/master/ocfweb/docs/docs/services/web.md#L16

the benefit is that the link gets automatically updated over time, plus it will fail the tests if the link is wrong (meaning we can't have broken links which go unnoticed)

Reply:
also tsunami is typo'd, and we shouldn't refer to "tsunami", but instead tell people to log in to "ssh.ocf.berkeley.edu" (the service CNAME)

Reply:
Should also be "in terminal" instead of "at terminal"